Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -2143,7 +2143,19 @@ type SecurityContextConstraints struct {
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
 	AllowPrivilegedContainer bool
+	// Required capabilities are defaulted into containers.
+	// Required capabilities will be enforced with the following rules
+	// 1.  RequiredCapabilities.Drop and AllowedCapabilities cannot conflict (you may not require
+	//		a cap to be dropped but also let it be requested)
+	// 2.  Capabilities in RequiredCapabilities.Add will be added to the container unless the container specifically
+	// 		requests it be dropped.
+	// 3.  Capabilities in RequiredCapabilities.Drop will be dropped from the container
+	// 4.  Containers that request to add a capability that is required to be dropped will fail
+	//		validation.
+	RequiredCapabilities Capabilities
```

Comment:
if we let pods drop capabilities that are in the requiredCapabilities.Add list, then "requiredCapabilities" is an odd name. "defaultCapabilities"?